managed-services/transactional-email/reference-content/understanding-tem-reputation-score.mdx

```diff
@@ -0,0 +1,75 @@
+---
+meta:
+  title: Understanding your reputation score with Transactional Email
+  description: This section provides information about your domains' reputation score
+content:
+  h1:  Understanding your reputation score with Transactional Email
+  paragraph: This section provides information about your domains' reputation score
+tags: transactional-email domain-reputation email-deliverability score reputation-score
+dates:
+  validation: 2023-07-28
+  posted: 2023-07-28
+categories:
+  - managed-services
+---
+
+This documentation provides advice on how to maintain a good [reputation](/managed-services/transactional-email/concepts/#reputation-score) score and helps you understand what you need to do, to have your emails delivered.
+
+<Macro id="iam-requirements" />
+
+
+## Checking your domain reputation
+
+Here is a list of some of the free tools you can use to check and monitor the reputation of your domains:
+
+- [Google Postmaster](https://postmaster.google.com/managedomains) tracks the reputation of your domain with Gmail users and allows you to get insights on how to improve your domain's reputation.
+- [MxToolBox](https://mxtoolbox.com/MXLookup.aspx) allows you to check for any blacklisting and deliverability issues that might be impacting your domains' reputation.
+- [BarracudaCentral](https://www.barracudacentral.org/lookups) keeps a database of IP addresses associated with known spammers and senders with good email practices. When you enter your IP or domain address, BarracudaCentral cross-references it with its database to determine your domain's reputation score.
+
+## How is domain reputation calculated?
+
+To manage your domain's reputation, you need to understand how it is calculated. The **factors** used to determine a domain's reputation score **vary from one service to another**.
+The only constant common denominator is that the **receiving platforms determine how your domain is used in your emails** and then **monitor how their users receive these emails**.
+Each domain has several reputations associated with different mail receivers. The reputations also depend on the volume of emails you send to specific receivers.
+
+## Factors that impact your domain reputation
+
+Many factors might affect your domain's reputation. The most common are the following:
+
+- Engagement: When determining domain reputation, engagement is one of the principal factors that comes into play. If recipients fail to open your emails or unsubscribe, email providers will interpret it as an indicator of bad reputation and this might impact your score.
```

Comment:
Does it mean if they don't open the emails and also don't unsubscribe? Or if they don't open the emails and/or they also unsubscribe?